packages/op-datasets/src/op_datasets/etl/intermediate/models/daily_address_summary.py

```diff
@@ -85,66 +112,60 @@
         alias="total_l2_gas_used_success",
         sql_expr=f"SUM(IF({TX_SUCCESS}, receipt_gas_used, 0))",
     ),
-    Expression(
-        alias="total_l2_gas_used_fail",
-        sql_expr=f"SUM(IF({TX_FAIL}, receipt_gas_used, 0))",
-    ),
     Expression(
         alias="total_l1_gas_used",
-        sql_expr="SUM(receipt_l1_gas_used)",
+        sql_expr=f"SUM(COALESCE(receipt_l1_gas_used, ({ESTIMATED_SIZE})))",
     ),
     Expression(
         alias="total_l1_gas_used_success",
-        sql_expr=f"SUM(IF({TX_SUCCESS}, receipt_l1_gas_used, 0))",
-    ),
-    Expression(
-        alias="total_l1_gas_used_fail",
-        sql_expr=f"SUM(IF({TX_FAIL}, receipt_l1_gas_used, 0))",
+        sql_expr=f"SUM(IF({TX_SUCCESS}, COALESCE(receipt_l1_gas_used, ({ESTIMATED_SIZE})), 0))",
     ),
     # Gas Fee Paid
     Expression(
         alias="total_gas_fees",
-        sql_expr=wei_to_eth(f"SUM({TOTAL_GAS})"),
+        sql_expr=wei_to_eth(f"SUM({TOTAL_GAS_FEES})"),
     ),
     Expression(
         alias="total_gas_fees_success",
-        sql_expr=wei_to_eth(f"SUM(IF({TX_SUCCESS}, {TOTAL_GAS}, 0))"),
-    ),
-    Expression(
-        alias="total_gas_fees_fail",
-        sql_expr=wei_to_eth(f"SUM(IF({TX_FAIL}, {TOTAL_GAS}, 0))"),
+        sql_expr=wei_to_eth(f"SUM(IF({TX_SUCCESS}, {TOTAL_GAS_FEES}, 0))"),
     ),
     # Gas Fee Breakdown
     Expression(
         alias="l2_contrib_gas_fees",
-        sql_expr=wei_to_eth(f"SUM({L2_CONTRIB_GAS})"),
+        sql_expr=wei_to_eth(f"SUM({L2_CONTRIB_GAS_FEES})"),
     ),
     Expression(
         alias="l1_contrib_gas_fees",
-        sql_expr=wei_to_eth(f"SUM({L1_CONTRIB_GAS})"),
+        sql_expr=wei_to_eth(f"SUM({L1_CONTRIB_GAS_FEES})"),
     ),
     Expression(
-        alias="l1_blobgas_contrib_gas_fees",
-        sql_expr=wei_to_eth(f"SUM({L1_BLOB_GAS_CONTRIB})"),
+        alias="l1_contrib_contrib_gas_fees_blobgas",
+        sql_expr=wei_to_eth(f"SUM({L1_CONTRIB_BLOB})"),
     ),
     Expression(
-        alias="l1_l1gas_contrib_gas_fees",
-        sql_expr=wei_to_eth(f"SUM({L1_L1_GAS_CONTRIB})"),
+        alias="l1_contrib_gas_fees_l1gas",
+        sql_expr=wei_to_eth(f"SUM({L1_CONTRIB_L1_GAS})"),
     ),
     Expression(
-        alias="l2_contrib_gas_fees_base_fees",
-        sql_expr=wei_to_eth(f"SUM({L2_CONTRIB_GAS} - {L2_CONTRIB_PRIORITY})"),
+        alias="l2_contrib_gas_fees_basefee",
+        sql_expr=wei_to_eth(f"SUM({L2_CONTRIB_BASE})"),
     ),
     Expression(
-        alias="l2_contrib_gas_fees_priority_fees",
+        alias="l2_contrib_gas_fees_priorityfee",
         sql_expr=wei_to_eth(f"SUM({L2_CONTRIB_PRIORITY})"),
     ),
+    Expression(
+        alias="l2_contrib_gas_fees_legacyfee",
```

Comment:
ok got it, we handle the legacyfee here, instead of as a definition. open to either strategy